src/docs/sdk/features.mdx

```diff
@@ -268,6 +268,36 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+
+### Scrubbing Sensitive Data
+
+HTTP Client Integrations record the URLs of HTTP requests. URLs can contain two categories of sensitive information:
+* Privacy related data like passwords, private keys, etc.
+* Personally identifiable information (PII) like email addresses, addresses, names, social security numbers, etc.
+
+#### Privacy Related Data
+
+Privacy related data from the `url` in breadcumbs and `$url` in span descriptions must always be scrubbed.
```

Comment:
I think it would be better to specify this so that future features are also included. 
What about 
```suggestion
The SDKs must scrub privacy-related data from URLs stemming from features that collect them automatically, such as HTTP breadcumbs and HTTP spans.
```